src/components/calcite-value-list/calcite-value-list.tsx

```diff
@@ -202,6 +203,38 @@ export class CalciteValueList {
 
   getItemData = getItemData.bind(this);
 
+  keyDownHandler = (event) => {
+    const handleElement = event.composedPath().find((item) => {
```

Comment:
question: composedPath doesn't work in edge. Is this polyfilled?

Reply:
yea it is...but testing in edge I'm seeing a different issue, the event.target is different because the shadow dom redirect doesn't happen...must fix.